includes/Story_Post_Type.php

```diff
@@ -110,7 +111,7 @@ public function init() {
 					'menu_name'                => _x( 'Stories', 'admin menu', 'web-stories' ),
 					'name_admin_bar'           => _x( 'Story', 'add new on admin bar', 'web-stories' ),
 				],
-				'menu_icon'             => 'dashicons-book',
+				'menu_icon'             => $this->get_post_type_icon( 'stories.svg' ),
```

Comment:
Why is this a public method accepting an arbitrary file name as an argument? Seems a bit overkill.

Also, I am wondering whether using `file_get_contents` is perhaps a bit too much.

IMO we can probably just use the hardcoded base64 string here. It would make things way easier as the icon never really changes.

If we want to, we could even do all of this (`base64_encode( SVG )`) when running `npm run workflow:build-plugin`.

Reply:
The reason why it is public and has a param on the method, is for unit testing and reusability.